### PR TITLE
Use line and column from node on DebugLexicalBlock.

### PR DESCRIPTION
**Pull request by sajjadmirzanv**
Old implementation had every DebugLexicalBlock hardcoded to (0, 0), but IR nodes know their source location so we may as well attach it to the DebugLexicalBlock.

### CODE REVIEW

**Comment by arcady-lunarg** [comment]
You should add a test that exercises this functionality.

**Comment by sajjadmirzanv** [comment]
> You should add a test that exercises this functionality.

I discovered that there was a bug in the test runner, which is why the existing nonsemantic tests were not already triggering this code. Running glslang -gVS on those files emits DebugLexicalLine, but it was missing when run through glslangtests. 

In TestFixture.h, the code was only generating debug info for the 2nd stage of compilation, from IR -> SPIRV. I enabled the flag so it also emits debug info during Source Language -> IR compilation. The result is that we get EOpScope nodes in the tree, so the GlslangToSpv converter can emit DebugLexicalBlock instructions. 

I had to make a few other small changes to some tests as well.

File: gtests/TestFixture.h [hunk 1]
```diff
@@ -262,6 +262,14 @@ class GlslangTest : public GT {
             shader.setAutoMapLocations(true);
             shader.setAutoMapBindings(true);
         }
+
+        if (enableDebug) {
+            shader.setDebugInfo(true);
+        }
+        if (enableNonSemanticShaderDebugInfo) {
```
Comment (arcady-lunarg) [inline gtests/TestFixture.h:269]: I like this forcing of correct usage. We might want to force `enableNonSemanticShaderDebugInfo` to set `enableDebug` at a lower level, but I will investigate that in a separate PR.

**Review by arcady-lunarg — APPROVED** [review]
LGTM, I think this looks reasonable.

**Pull request merged**